lunarlandercolab.py

```diff
@@ -348,7 +364,25 @@ def runME(runnum, game, sequence_len,
 
         game.run(cur_agent)
         # Keep track of changes here...
-        feature_map.add(cur_agent)
+        did_add = feature_map.add(cur_agent)
+
+        # On each add (i.e. data change) update the checkpoint file.
+        if did_add:
+            # Update the main checkpoint file.
+            checkpoint_file = os.path.join(CHECKPOINTS_DIR, CHECKPOINT_FILE_NAME + "_{}_latest.{}".format(session_checkpoint_time, CHECKPOINT_EXTENSION))
+            with open(checkpoint_file, "wb") as f:
+                pickle.dump(feature_map, f)
+            num_checkpoints += 1
+            # print("current_time: ", last_checkpoint_time)
+            # If it's been enough time since the last incremental checkpoint, create one
+            if num_checkpoints % CHECKPOINT_FREQUENCY == 0:
+                # save a secondary checkpoint
+                checkpoint_file = os.path.join(CHECKPOINTS_DIR,
```

Comment:
NOTE: use number of individuals count as the checkpoint ID for the given session.